base/common/system.py

```diff
@@ -48,6 +48,11 @@ def _remove_heading_from_df_output(df_output: bytes) -> int:
         LOG.info(f"obtaining free space on bu hdd with command: {' '.join(command)}. Received {free_space_on_bu_hdd}")
         return free_space_on_bu_hdd
 
+    @staticmethod
+    def system_clock_synchronized_with_ntp() -> bool:
+        timedate_status_raw = subprocess.check_output("timedatectl")
+        return "System clock synchronized: yes" in timedate_status_raw.decode()
```

Comment:
If the output of timedatectl should ever change, this will always return `False`. Maybe look for `"System clock synchronized: yes"` as well and if both are not found, raise an Exception.

Reply:
Maybe consider writing another function `wait_for_ntp_update()` calling `system_clock_synchronized_with_ntp()` in a loop which raises an `TimeoutError` if a timeout occurs. The latter can become protected then.